README.md

```diff
@@ -183,15 +177,15 @@ You likely want to add this to the **Get Data** section, so your modification wi
 <toolbox monitor="true">
   <section id="getext" name="Get Data">
     <!-- Add below line to your file -->
-    <tool file="irida-galaxy-importer/irida_import/irida_import.xml" />
+    <tool file="irida_import/irida_import.xml" />
```

Comment:
Shouldn't this be `irida_import/irida_import/irida_import.xml`? Or does the XML file get generated one directory up now?

Reply:
Do to changes introduced by upgraded to python 3 the tool is installed differently. Previously you would install the `irida_import` folder and your xml and python files lived in the same directory. Now you the folder structure directly mimics the repo, so the generated xml is one directory above the sample xml.